__tests__/.loadtestrc

```diff
@@ -0,0 +1,7 @@
+limits:
```

Comment:
This file can be placed in a `test` directory, I guess?

Reply:
As you want…